apps/client/src/assets/ProgressBar.tsx

```diff
@@ -0,0 +1,17 @@
+interface Colors {
+  color: string;
+}
+
+const ProgressBar = ({ color }: Colors) => (
```

Comment:
```suggestion
interface ProgressBarProps {
  color: string;
}

const ProgressBar = ({ color }: ProgressBarProps) => (
```

props interface이니 명칭을 이렇게 바꾸는게 어떨까요?

Reply:
수정하였습니다! [8e785bf](https://github.com/themoment-team/hellogsm-front-24/pull/26/commits/8e785bf939e135d6e5e6930e306ed9766ffa6b5e)